patch-setup-config.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
```

Comment:
I think we better do this in Haskell. There is a function called `fixFile` that is used for this purpose.